README.md

```diff
@@ -2,7 +2,42 @@
 
 ## Running project locally
 
-Run `npm run start:consulting` or `npm run start:labs` to start a corresponding dev server. Navigate to http://localhost:4200/ or use localhost preview in Storyblok panel. On the localhost the app will automatically reload if you change any of the source files, in the Storyblok panel you need to refresh the page manually.
+Prerequisites: Node and npm.
```

Comment:
Please add links for lazy folks like me

Reply:
Done. No need to mention NPM as it's a part of NodeJS installation package.